commands/build-tokenizer-states.js

```diff
@@ -0,0 +1,53 @@
+const https = require('https');
+const fs = require('fs');
+
+const OUTFILE = require('path').resolve(__dirname, '../src/tokenizer-states.ts');
+const SYNTAX_URL = 'https://raw.githubusercontent.com/w3c/html/master/sections/syntax.include';
```

Comment:
Should we use a specific SHA / tag?

Reply:
I don't think so, this is a command that you must run manually.

It might be nice to include the SHA in the generated output though.